index.bs

```diff
@@ -1017,6 +1017,20 @@ For example,
 is more readable than
 <code highlight="js">window.scrollBy(50, 0)</code>.
 
+<h3 id="constructors">Make non-critical function parameters optional</h3>
+
+Parameters that are not strictly necessary should be made optional.
+Making arguments optional with reasonable defaults reduce overhead when the function is used,
+and increases user-friendliness.
+
+Optional parameters should treat empty or undefined as equal to the omission of the parameter.
+For example, given a function foo() which takes a single optional parameter,
```

Comment:
`<code highlight="js">` around code examples.

Also, can we make it a specific, real-world example, e.g.

```html
<p class="example">For example, 
<code highlight="js">element.scrollIntoView(false, {});</code> 
is equivalent to 
<code highlight="js">element.scrollIntoView(false);</code>,
and
<code highlight="js">element.addEventListener("click", onClick, undefined);</code>
is equivalent to 
<code highlight="js">element.addEventListener("click", onClick)</code>.
</p>
```

Reply:
Addressed in 0f0ae